docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -162,6 +162,10 @@ In later tutorials, you learn how to interact with a smart contract from the bro
 
   The `src/index.ts` file contains all of the exports you want to make available for consumption from outside your smart contract project, such as from a UI.
 
+  :::info 
```

Comment:
@Violet-Bora-Lee  I don't get this error message when I run through this tutorial.

What are the steps you are following to get the error message?

Reply:
If you run an already-done tutorial, it won't show up. But it will show up if you run it from the top in turn.

Reply:
I was wondering if I should include this information, but I stopped here to read the error and thought for a short time about what I did wrong with the error, so I thought it would be better to provide readers with information.

Reply:
If you think it's too small an error, I'll take this part out and fix the pull request. 😉

Reply:
let me run through it as you did, and we'll include as appropriate, thank you!

Reply:
@Violet-Bora-Lee  let's remove since I followed step by step and did not encounter error (Maybe you ran code before the tutorial explicity stated to run code?)

Thank you for your attention!